playbooks/roles/nginx/files/sites/opencraft-tasks/tasks.opencraft.com.conf

```diff
@@ -0,0 +1,60 @@
+server {
```

Comment:
@Agrendalath, the `tasks.opencraft.com` domain is pointed to a different VM. So we should not include this in this PR. Ditto for other OpenCraft sites not hosted on the old opencraft.com VM.